test/lib/maintenance_tasks/cli_test.rb

```diff
@@ -85,25 +85,31 @@ class CLITest < ActiveSupport::TestCase
       end
     end
 
-    test "`help perform` loads all tasks and displays them" do
+    test "`help perform` invites the user to use `maintenance_tasks list` with proper rewrapping" do
+      expected_output = <<~OUTPUT.indent(2)
+        `maintenance_tasks perform` will run the Maintenance Task specified by the [TASK NAME] argument.
+
+        Use `maintenance_tasks list` to get a list of all available tasks.
+      OUTPUT
+
+      assert_output(Regexp.union(expected_output)) do
+        CLI.start(["help", "perform"])
+      end
+    end
+
```

Comment:
This test is probably unnecessary. (It tests Thor's standard behaviour, not anything related to maintenance_tasks' code.)

Remove?

Reply:
Is this comment about the `help perform` test or the `list` one? I'd prefer to keep the `list` one, but fine with dropping the `help perform` one since we're matching on a simple string.